mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/plan/synchronization/spi/impl/AsyncIndexingPlanSynchronizationStrategy.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.plan.synchronization.spi.impl;
```

Comment:
I think there's a mistake here, the package should be `.impl`, not `.spi.impl`...